app/src/main/java/woowacourse/paint/PaintCanvasView.kt

```diff
@@ -0,0 +1,70 @@
+package woowacourse.paint
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Path
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+
+class PaintCanvasView(context: Context, attrs: AttributeSet) : View(context, attrs) {
+    private var selectedColor = Color.parseColor(PaintColor.RED.value)
+    private var selectedStrokeWidth = 10F
+
+    private var path = Path()
+    private var paint = createPaintWith(selectedColor, selectedStrokeWidth)
+    private val canvasData = mutableMapOf(Pair(path, paint))
```

Comment:
```suggestion
    private val canvasData = mutableMapOf(path to paint)
```
이렇게도 간단히 나타낼 수 있답니당

Reply:
```suggestion
    private val canvasData = mutableListOf<Pair<Path, Paint>>()
```

1) 해시 불일치의 위험성 

Path 를 key 값으로 지정해주었는데요.  
 Path 의 내부 field 값이 바뀌면 hashCode 값이 바뀔 위험이 있어 해시 충돌이 날 수 있어요 😨  

List 방식으로 수정해주심 좋을 것 같습니다 ㅎㅎ

2) 초기값을 비워주세요!

초기값을 설정해주셔서 onDraw() 호출 시 불필요한 빈 경로가 그려지고 있습니다